### PR TITLE
[fix] MediaObject height

### PR DESCRIPTION
**Pull request by richardtagger**
Fixing the height of MediaObject

## Pull request checklist

* [x] Component `README.md` file is up-to-date.
* [x] Component is unit tested.

### CODE REVIEW

**Comment by unindented** [comment]
Why did a bunch of new reference files unrelated to your change appear with the PR?

**Comment by richardtagger** [comment]
IDK - I changed the code I changed, ran the tool and that's how I got?

**Comment by richardtagger** [comment]
Ok done, can you take a look again, please

File: src/components/MediaObject/__snapshots__/MediaObject.test.tsx.snap [hunk 1]
```diff
@@ -173,7 +165,6 @@ exports[`<MediaObject /> with title content matches its snapshot 1`] = `
     gutterSize="medium"
   >
     <FixedGridColumn
-      className="y-media-object__size-medium--image"
```
Comment (unindented) [inline src/components/MediaObject/__snapshots__/MediaObject.test.tsx.snap]: Is there anything we can delete from the CSS file now that these class names are gone?

Comment (swese44) [reply src/components/MediaObject/__snapshots__/MediaObject.test.tsx.snap]: yes there's related CSS. but we need to hear from @adrianmg as the original 33px height was deliberate

**Comment by richardtagger** [comment]
@adrianmg fixed things on his branch so closing this

**Pull request closed without merging**